Expecta/EXPExpect.m

```diff
@@ -43,12 +43,16 @@ - (instancetype)initWithActualBlock:(id)actualBlock testCase:(id)testCase lineNu
     self.timeout = [Expecta asynchronousTestTimeout];
     self.lineNumber = lineNumber;
     self.fileName = fileName;
+    self.completed = NO;
   }
   return self;
 }
 
 - (void)dealloc
 {
+  if (!self.completed) {
+    EXPFail(self.testCase, self.lineNumber, self.fileName, @"Expectation is not complete");
```

Comment:
I'd change the failure text to something more elaborated, such as `Called expect() without running a matcher`.

Reply:
done